sanic/blueprint_group.py

```diff
@@ -130,6 +151,15 @@
         :param item: New `Blueprint` object.
         :return: None
         """
+        if self._version and item.version and self._version != item.version:
```

Comment:
I would add an `@property` for `BlueprintGroup.version` and `BlueprintGroup.strict_slashes`

Reply:
Done.